draft-irtf-cfrg-opaque.md

```diff
@@ -1070,11 +1235,47 @@ such that the following conditions are met:
 - The MHF has fixed parameters, chosen by the application, and implements the
   interface in {{dependencies}}. Examples include Argon2 {{?I-D.irtf-cfrg-argon2}},
   scrypt {{?RFC7914}}, and PBKDF2 {{?RFC2898}} with fixed parameter choices.
-- EnvelopeMode value is as defined in {{credential-storage}}, and is one of
+- EnvelopeMode value is as defined in {{client-credential-storage}}, and is one of
   `base` or `custom_identifier`.
 - The Group mode identifies the group used in the OPAQUE-3DH AKE. This SHOULD
   match that of the OPRF. For example, if the OPRF is OPRF(ristretto255, SHA-512),
   then Group SHOULD be ristretto255.
+  
+To generate the private key in the `Internal` mode, OPAQUE-3DH implements `DeriveKeyPair(seed)` as follows:
```

Comment:
I know I was the one who suggested this be down in {{configurations}}, but now I'm second guessing that. Perhaps these should be kept in the external mode section? @kevinlewi, what do you think? (Also, is the generator G defined anywhere?)

Reply:
external or internal?

Reply:
It's only used in the internal mode, it makes total sense to define it there, yes.